### PR TITLE
NET-4884 - Terminating gateway tests for namespaces & partitions

### PR DESCRIPTION
**Pull request by curtbushko**
### Description

Manual sync of gateway tests.

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] appropriate backport labels added
* [ ] not a security concern

### CODE REVIEW

File: test/integration/consul-container/test/gateways/terminating_gateway_test.go [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+			require.Contains(r, out, fmt.Sprintf("FORTIO_NAME=%s", serviceName))
+			t.Logf("successfuly messaged %s", serviceName)
+		} else {
+			require.Error(r, err)
+			require.Contains(r, err.Error(), "exit code 52")
+		}
+	})
+}
+
+func createNamespace(t *testing.T, consulClient *api.Client, namespace string) {
```
Comment (wilkermichael) [inline test/integration/consul-container/test/gateways/terminating_gateway_test.go:191]: I think actually this should be in an enterprise only file, you might want to refactor this?

**Review by wilkermichael — COMMENTED** [review]
Just a quick thought on a namespace function in here. Also be tactical with the use of namespace/partition in CE tests I think?

File: test/integration/consul-container/test/gateways/terminating_gateway_test.go (hunk 1 above)
Comment (curtbushko) [reply test/integration/consul-container/test/gateways/terminating_gateway_test.go:191]: Sure!

Comment (curtbushko) [reply test/integration/consul-container/test/gateways/terminating_gateway_test.go:191]: I've moved it out and into the other ENT PR.

**Review by wilkermichael — APPROVED** [review]

**Pull request merged**